fileWithPRComments.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
```

Comment:
![Codacy](https://app.staging.codacy.org/assets/images/favicon.png) Issue found: [Move the invocation into the parens that contain the function.](https://app.staging.codacy.org/manual/stagingcodacytestuser/CheckPrCommentsPublicProjectStaging/pullRequest?prid=10014)